builder/img/protocjs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# install googleapis common protos
+shasum="9584b7ac21de5b31832faf827f898671cdcb034bd557a36ea3e7fc07e6571dcb"
+curl -fSLo /tmp/common-protos.tar.gz "https://github.com/googleapis/googleapis/archive/common-protos-1_3_1.tar.gz"
```

Comment:
Is this needed?

Reply:
Hmm, that's used for the `/usr/local/include/google/api/...` args to `protoc` above, though those don't appear required anywhere so I think this is just dead code (there would be an error somewhere in the build process if it is in fact required once it's removed, so I'll just remove it).

Reply:
Yeh I think we removed the `google/api/...` imports